server.js

```diff
@@ -16,9 +17,8 @@ app.get('/', (request, response) => {
 });
 
 app.get('/api/v1/books', (request, response) => {
-  const books = app.locals.books;
-
-  response.status(200).json({ books });
+  queries.getAll().then(data => response.send(data))
+  .catch(error => response.status(500).json({ error }))
```

Comment:
Awesome job Christine. We'll probably want to add some more error handling down the line, but this is a great start!